src/bmi_lgar.cxx

```diff
@@ -83,7 +83,40 @@ Update()
     std::cerr<<"|****************** LASAM BMI Update... ******************|\n";
     std::cerr<<"---------------------------------------------------------\n";
   }
- 
+
+  double mm_to_cm = 0.1; // unit conversion
+
+  if (state->lgar_bmi_params.is_invalid_soil_type) {
+    // add to mass balance accumulated variables
+    state->lgar_mass_balance.volprecip_cm  += state->lgar_bmi_input_params->precipitation_mm_per_h * mm_to_cm;
+    state->lgar_mass_balance.volin_cm       = 0.0;
+    state->lgar_mass_balance.volon_cm       = 0.0;
+    state->lgar_mass_balance.volend_cm      = state->lgar_mass_balance.volstart_cm;
+    state->lgar_mass_balance.volAET_cm      = 0.0;
+    state->lgar_mass_balance.volrech_cm     = 0.0;
+    state->lgar_mass_balance.volrunoff_cm  += state->lgar_bmi_input_params->precipitation_mm_per_h * mm_to_cm;
```

Comment:
is volrunoff and volQ the same thing?  If not, this looks like a mass balance  logical error since the precipitation is being accounted for twice.

Reply:
currently they are the same, once we merge LGARTO, they will be different